src/server/ssrServer.js

```diff
@@ -57,6 +59,7 @@ export function createApp({ enablePostToModuleRoutes = false } = {}) {
 
   app.use('/_/static', express.static(path.join(__dirname, '../../build'), { maxage: '182d' }));
   app.get('*', addCacheHeaders);
+  app.use(pwaBasePath, pwaRouter);
```

Comment:
for consistency I'd anticipate
```suggestion
  app.use('/_/pwa', pwaRouter);
```
would help see which routes are handling a request and branch from here rather than having to check another file to see what the path was (of each `.use`/`.get`)